### PR TITLE
[ROCm] unskip FFT tests

### PR DESCRIPTION
**Pull request by micmelesse**
This pr enables a lot of fft tests on rocm, see the full list here https://github.com/ROCmSoftwarePlatform/pytorch/issues/924. 

After enabling the tests we found that 3 tests , test_reference_1d, test_reference_nd and test_fn_grad have issue.  We skip those tests on ROCM in this pr as well. We will address those skipped tests in a subsequent pr.

### CODE REVIEW

**Comment by jeffdaily** [comment]
@malfet or @seemethere or delegate, would you be able to merge this?  It unskips many FFT tests for ROCm.

**Review by malfet — APPROVED** [review]
LGTM, but looks like it skips the entire `test_fn_grad`

File: test/test_ops_gradients.py [hunk 1]
```diff
@@ -113,6 +113,7 @@ def _skip_helper(self, op, device, dtype):
             self.skipTest("Skipped! Complex autograd not supported.")
 
     # Tests that gradients are computed correctly
+    @skipCUDAIfRocm
```
Comment (malfet) [inline test/test_ops_gradients.py:116]: @jeffdaily just checking: are you ok with that?

**Pull request closed without merging**